lib/build/builder.js

```diff
@@ -4,16 +4,14 @@ import {CompositeDisposable, Range} from 'atom'
 import fs from 'fs-extra'
 import path from 'path'
 import temp from 'temp'
+import {isValidEditor} from './../utils'
 
 class Builder {
-  constructor (goconfig) {
+  constructor (goconfig, linter) {
```

Comment:
Is there a way to remove the linter dependency and replace it with some shim that lets us adjust approach in the future (e.g. diagnostics) in one place rather than many?